pixi/backend/api.js

```diff
@@ -56,6 +57,28 @@ const isCacheUrl = (urlString, cachesList) => {
   return !!matchedCache;
 };
 
+const getComponentVersions = async () => {
```

Comment:
Can we move the check into linter?

Reply:
Moved here ampproject/amp-toolbox/pull/1165.